package.json

```diff
@@ -17,7 +17,8 @@
     "start:syncserver": "cross-env DEBUG='WebsocketServer' pnpm -F @automerge/example-sync-server start",
     "preinstall": "npx only-allow pnpm",
     "repocheck": "manypkg check",
-    "lint": "eslint --ext .ts .",
+    "format": "prettier --write . | dev-null",
```

Comment:
`prettier --write` outputs to the console the name of every single file it looks at, whether it modifies it or not. That's a lot of noise, so we pipe that output to `dev/null` (or `NUL` on windows). 

Errors still are reported to the console.